clean_db.sh

```
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# ========================
+# CARGAR VARIABLES DESDE .env
+# ========================
+if [ -f .env ]; then
+    export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)